PAM_app/app/src/main/java/com/example/pam_app/foodAndDrinks/FoodAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.pam_app.foodAndDrinks
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.RecyclerView
+import com.example.pam_app.R
+
+internal class FoodAdapter(private var foodList: List<FoodModel>) : RecyclerView.Adapter<FoodAdapter.MyViewHolder>() {
+
+    internal inner class MyViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+        var image: ImageView = view.findViewById(R.id.imageFood)
+        var name: TextView = view.findViewById(R.id.textFoodName)
+        var price: TextView = view.findViewById(R.id.textPrice)
+    }
+
+    @NonNull
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MyViewHolder {
+        val itemView = LayoutInflater.from(parent.context).inflate(R.layout.list_item_grid_food, parent, false)
```

Comment:
Change init of recycler view items to android view binding https://developer.android.com/topic/libraries/view-binding

You can consult example from my sample repository.